app/api/article.tsx

```diff
@@ -78,6 +80,10 @@ class ArticleAPI extends PlutoAxios {
       },
       cancelToken: cancelTokenSource.token,
     });
+    var endTime = performance.now();
+    var diffTime = endTime - startTime;
+    measureTiming("article-api", "getPapers", diffTime);
```

Comment:
I think we don't need to track this value because it depends on user's internet status rather than our code quality.